tools/index.qmd

```diff
@@ -2,6 +2,41 @@
 title: "Stan Toolkit"
 page-layout: full
 toc-location: right
+
+aliases:
+ - ../interfaces/loo.html 
+ - ../interfaces/rstanarm.html 
+ - ../interfaces/pystan.html 
+ - ../interfaces/mathematica-stan.html 
+ - ../interfaces/rstantools.html 
+ - ../interfaces/math.html 
```

Comment:
Similarly, if we really wanted to, this could link to `/tools/#developer-tools-and-apis`